File: two sum/Brute Force/twoSum.cpp

class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        vector<int> output;
        for(auto it =  nums.begin(); it != nums.end(); it++) {
            auto it1 = find(it+1, nums.end(), target - *it);
            if(it1 != nums.end()) {
                output.push_back(it-nums.begin());
                output.push_back(it1-nums.begin());
        }
        }
        return output;
    }
};

File: two sum/Hash Map Solution/twoSum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        hashMap = {}
        for index,value in enumerate(nums):
            diff = target - value
            if diff in hashMap:
                return [hashMap[diff],index]
            hashMap[value] = index
        return 